ui/src/app/settings/components/project-role-jwt-tokens/project-role-jwt-tokens.tsx

```diff
@@ -39,11 +39,11 @@ export const ProjectRoleJWTTokens = (props: ProjectRoleJWTTokensProps) => {
                             {props.tokens.map((jwtToken: JwtToken) => (
                                 <div className='argo-table-list__row' key={`${jwtToken.iat}`}>
                                     <div className='row'>
-                                        <div className='columns small-3'>{jwtToken.iat}</div>
+                                        <div className='columns small-3'>{jwtToken.id}</div>
```

Comment:
The id field is empty for tokens that were created before this change. I think we can handle it in API server:

https://github.com/argoproj/argo-cd/blob/63da539bc39e245da734d7deb5029864a71c312a/server/project/project.go#L212 

E.g.:

```go
func (s *Server) Get(ctx context.Context, q *project.ProjectQuery) (*v1alpha1.AppProject, error) {
	if err := s.enf.EnforceErr(ctx.Value("claims"), rbacpolicy.ResourceProjects, rbacpolicy.ActionGet, q.Name); err != nil {
		return nil, err
	}
	proj, err := s.appclientset.ArgoprojV1alpha1().AppProjects(s.ns).Get(q.Name, metav1.GetOptions{})
	if err != nil {
		return nil, err
	}
	for i, role := range proj.Spec.Roles {
		for j, token := range role.JWTTokens {
			if token.ID == "" {
				token.ID = strconv.FormatInt(token.IssuedAt, 10)
				role.JWTTokens[j] = token
			}
		}
		proj.Spec.Roles[i] = role
	}
	return proj, err
}
```